src/abstractdataframe/abstractdataframe.jl

```diff
@@ -296,7 +296,7 @@ tail(df)
 """
 (head, tail)
 
-# get the structure of a DT
+# get the structure of a DF
```

Comment:
We'll still need an ugly cleanup commit anyways, so I don't know if it's a big deal. I'll squash my additional changes once we're happy with the changes.